src/applications/disability-calculator/components/RatingRow.jsx

```diff
@@ -52,6 +52,7 @@ const RatingRow = ({
         <div>
           <button
             type="button"
+            aria-label={`Remove disability row ${indx + 1}`}
```

Comment:
Let's change the word "Remove" to "Delete" to match our visible label for [WCAG 2.1 Success Criterion 2.5.3 Label in Name](https://www.w3.org/TR/WCAG21/#label-in-name) best practice. After that text item is changed, this is ready to go!

I tested this in the following screen readers:
* NVDA + Firefox, Chrome
* JAWS + IE11, Chrome
* VoiceOver + Safari